ephios/plugins/guests/templates/guests/event_detail.html

```diff
@@ -0,0 +1,10 @@
+{% extends "core/event_detail.html" %}
+{% load bootstrap %}
+{% load rich_text %}
+{% load guardian_tags %}
+{% load i18n %}
+
+{% block content %}
+    <h3 class="text-info">{{ object.as_participant }}</h3>
```

Comment:
this seems a bit weird with just the name as such a big heading

Reply:
I can make it smaller :D
Was just simple this way...